definitions/ext-flex_host/definition.yml

```diff
@@ -0,0 +1,28 @@
+domain: EXT
+type: FLEX_HOST
+synthesis:
+  name: displayName
+  identifier: displayName
+  encodeIdentifierInGUID: true
+  conditions:
+  - attribute: eventType
+    value: FlexHostSample
+  tags:
+    arch:
+      multiValue: false
+    coreCount:
+      multiValue: false
+    fullHostname:
+      multiValue: false
+    processorCount: 
+      multiValue: false
+    localIp: 
+      multiValue: false
+    shortHostname: 
+      multiValue: false
+      entityTagName: hostname
+dashboardTemplates:
+  newRelic:
+    template: dashboard.json
+configuration:
+  alertable: true
```

Comment:
is the default ttl good enough?

Reply:
@naxhh have set it to QUARTERLY now, this should be sufficient.